### PR TITLE
Adding Spiral Framework 3.0 support

### PR DESCRIPTION
**Pull request by msmakouz**
Test errors with `prefer-lowest` are related to: 
https://github.com/spiral/nyholm-bridge/pull/6

### CODE REVIEW

File: tests/HttpTrait.php [hunk 1]
```diff
@@ -49,6 +49,17 @@ private function request(
...
 
-        return new ServerRequest([], [], $uri, $method, 'php://input', $headers, $cookies, $query);
+        $request = new ServerRequest(
+            method: $method,
+            uri: $uri,
+            headers: $headers,
+            body: 'php://input'
+        );
+
+        $request = $request
```
Comment (butschster) [inline tests/HttpTrait.php:59]: Here can be return

File: .github/workflows/phpunit.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+name: phpunit
+
+jobs:
+  phpunit:
+    uses: spiral/gh-actions/.github/workflows/phpunit.yml@master
+    with:
+      os: >-
+        ['ubuntu-latest']
+      php: >-
+        ['8.1']
```
Comment (butschster) [inline .github/workflows/phpunit.yml:16]: Add php 8.2

File: tests/HttpTrait.php (hunk 1 above)
Comment (msmakouz) [reply tests/HttpTrait.php]: Done

File: .github/workflows/phpunit.yml (hunk 2 above)
Comment (msmakouz) [reply .github/workflows/phpunit.yml]: Added

**Review by butschster — APPROVED** [review]

**Pull request merged**